### PR TITLE
Outlook October changes

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810251938378161-518%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=zTNoQIQVpv1n2gshXfceN3PhQQ0lEBKkW%2FbzSsq8nXg%3D&st=2018-10-25T19%3A34%3A53Z&se=2018-11-25T19%3A39%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810252159548060-518%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eex9xruGvzroDu2FLHiSbBXbEzQH2u4owj5LkKeKyKM%3D&st=2018-10-25T21%3A56%3A12Z&se=2018-11-25T22%3A01%3A12Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by ElizabethSamuel-MSFT** [comment]
Related PRs: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30019, https://github.com/OfficeDev/office-js-docs-reference/pull/227

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810252327499508-518%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=lrfUil3LIiBRfJM%2B%2Bm68214xWlV6nan1v25KMPx38hM%3D&st=2018-10-25T23%3A24%3A06Z&se=2018-11-25T23%3A29%3A06Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810291607500823-518%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=htdkVrItRNDhn3iRj7GQDXWYxDo5jgbKirRadwfJfWY%3D&st=2018-10-29T16%3A04%3A11Z&se=2018-11-29T16%3A09%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md [hunk 1]
```diff
@@ -1335,6 +1354,112 @@ Office.context.mailbox.item.displayReplyForm(
...
+|[Applicable Outlook mode](https://docs.microsoft.com/outlook/add-ins/#extension-points)|Compose or read|
+
+##### Returns:
+
+Type:
+[AttachmentContent](/javascript/api/outlook/office.attachmentcontent)
+
+##### Example
+
+```
```
Comment (kbrandl) [inline docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md]: Add language to this code sample (```javascript```) to enable code highlighting on the page.

File: docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md [hunk 2]
```diff
@@ -11,14 +11,21 @@ The Preview Requirement set includes all of the features of [Requirement set 1.7
...
 - [Office.context.mailbox.item.getSharedPropertiesAsync](office.context.mailbox.item.md#getsharedpropertiesasyncoptions-callback) - Added a new method that gets an object which represents the sharedProperties of an appointment or message item.
+- [Office.context.mailbox.item.internetHeaders](office.context.mailbox.item.md#internetheaders-internetheadersjavascriptapioutlookofficeinternetheaders) - Added a new property that represents the internet headers on a message item.
 - [Office.context.auth.getAccessTokenAsync](https://docs.microsoft.com/office/dev/add-ins/develop/sso-in-office-add-ins#sso-api-reference) - Added access to `getAccessTokenAsync`, which allows add-ins to [get an access token](https://docs.microsoft.com/outlook/add-ins/authenticate-a-user-with-an-sso-token) for the Microsoft Graph API.
+- [Office.MailboxEnums.AttachmentContentFormat](/javascript/api/outlook/office.mailboxenums.attachmentcontentformat) - Added a new enum that specifies the formatting that applies to an attachment's content.
```
Comment (kbrandl) [inline docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md]: This link 404s in your branch...but works okay in **master**, so presumably everything will work fine when your branch is merged to master.

File: docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md [hunk 3]
```diff
@@ -11,14 +11,21 @@ The Preview Requirement set includes all of the features of [Requirement set 1.7
...
+- [Office.MailboxEnums.AttachmentStatus](/javascript/api/outlook/office.mailboxenums.attachmentstatus) - Added a new enum that specifies whether an attachment was added to or removed from an item.
```
Comment (kbrandl) [inline docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md]: Same comment here -- this link 404s in your branch...but works okay in **master**, so presumably everything will work fine when your branch is merged to master.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/office.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-outlook-october-changes/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C31%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810311849008110-518%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9UW9yOTuk96BzL7N69KwBeorY4SkGv7m6vB8%2FexL48k%3D&st=2018-10-31T18%3A45%3A40Z&se=2018-12-01T18%3A50%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md]: The -reference branch was deleted as the change had been merged to master so it's a good sign. Thanks for checking!

File: docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md (hunk 3 above)
Comment (ElizabethSamuel-MSFT) [reply docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md]: ditto - this is good as -reference changes have been merged to master.

File: docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md]: Addressed in PR 532.

**Pull request merged**